test/e2e/data/infrastructure-docker/v0.3/bases/cluster-with-kcp.yaml

```diff
@@ -67,10 +67,12 @@ spec:
       nodeRegistration:
         criSocket: unix:///var/run/containerd/containerd.sock
         kubeletExtraArgs:
+          cgroup-driver: cgroupfs
```

Comment:
Should we add a short comment above all of those for future-us that we pin this because we use this template only with Kubernetes <=1.23 for clusterctl ugprade tests?

Reply:
And that in other cases ClusterClass is doing it for us automatically.

Btw what about 1.0?